Build fails

**Describe the bug**
Building the project fails.

From the top-level directory, running `make` gives:

```bash
newton-irPass-ipsaBackend.c: In function 'createInstructionsForSignal':
newton-irPass-ipsaBackend.c:1242:18: error: variable 'parallel' set but not used [-Werror=unused-but-set-variable]
 1242 |             bool parallel = false;
      |                  ^~~~~~~~
newton-irPass-ipsaBackend.c:1224:17: error: variable 'shorterLength' set but not used [-Werror=unused-but-set-variable]
 1224 |             int shorterLength = 0;
      |                 ^~~~~~~~~~~~~
newton-irPass-ipsaBackend.c: At top level:
cc1: error: unrecognized command line option '-Wno-gnu-designator' [-Werror]
cc1: all warnings being treated as errors
make[1]: *** [Makefile:273: newton-irPass-ipsaBackend.o] Error 1
make[1]: Leaving directory '/home/blackgeorge/Noisy-lang-compiler/src/newton'
make: *** [Makefile:17: newton] Error 2
```

**To Reproduce**
Steps to reproduce the behavior:
1. `make`
2. See error.

**Host OS :**
 - OS: Ubuntu 18.04

--- a/src/newton/newton-irPass-ipsaBackend.c
+++ b/src/newton/newton-irPass-ipsaBackend.c
@@ -1221,32 +1221,33 @@ createInstructionsForSignal(State * N, Signal * signal, int * instructionIndex, 
             generateRWListsForSignalLists(N, randomGroup1, randomGroup2, astNodeStrings, instructionIndex, i2cAddressList1, registerAddressList1, writeValueList1, rwList1, i2cAddressList2, registerAddressList2, writeValueList2, rwList2, &numberOfSensorInterfaceCommands1, &numberOfSensorInterfaceCommands2, &numberOfWriteCommands1, &numberOfWriteCommands2, physicalGroupNumberList1, physicalGroupNumberList2, dimIndexList1, dimIndexList2);
 
             int j = 0;
-            int shorterLength = 0;
+            // int shorterLength = 0; // TODO: Check if variable is useful.
             int difference = 0;
             if(numberOfWriteCommands1 > numberOfWriteCommands2)
             {
                 j = numberOfWriteCommands1;
-                shorterLength = numberOfWriteCommands2;
+                // shorterLength = numberOfWriteCommands2;
                 difference = numberOfWriteCommands1 - numberOfWriteCommands2;
 
             } else if(numberOfWriteCommands1 < numberOfWriteCommands2) {
                 j = numberOfWriteCommands2;
-                shorterLength = numberOfWriteCommands1;
+                // shorterLength = numberOfWriteCommands1;
                 difference = numberOfWriteCommands2 - numberOfWriteCommands1;
             } else {
-                shorterLength = numberOfWriteCommands1;
+                // shorterLength = numberOfWriteCommands1;
                 j = numberOfWriteCommands1;
                 difference = 0;
             }
 
-            bool parallel = false;
+            // TODO: Check if code necessary.
+            // bool parallel = false;
 
-            if(numberOfWriteCommands1 == 0 || numberOfWriteCommands2 == 0)
-            {
-                parallel = false;
-            } else {
-                parallel = true;
-            }
+            // if(numberOfWriteCommands1 == 0 || numberOfWriteCommands2 == 0)
+            // {
+            //     parallel = false;
+            // } else {
+            //     parallel = true;
+            // }
         
 
             for(int i=0; i<j; i++)

